### ISSUE TITLE
setup module: add public ed25519 host key to facts

### ISSUE THREAD

**Issue by nusenu** [ansible/ansible-modules-core#3290]
##### Issue Type:
- Feature Idea
##### Plugin Name:

setup
##### Ansible Version:

```
1.9.4
```
##### Summary:

add public ed25519 host key to facts
##### Steps To Reproduce:

```
ansible -m setup target 
```
##### Expected Results:

Output should contain fact for ed25519 host key
##### Actual Results:

Output does not contain ed25519 host key

**Comment by ansibot** [comment]
@grossws ping, this issue is waiting for your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@grossws, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by robinro** [comment]
Ansible 2.0 has the ed25519 key in it's facts, since ansible/ansible@e7846343e57691f827623047b140ccbe938a13eb.  I doubt that this is important enough to be backported to 1.9, so I'd recommend to close this issue.
wontfix @grossws @bcoca

**Comment by bcoca** [comment]
closing as per above
